libs/vre/shared/app-ontology/src/lib/property-info/property-info.component.ts

```diff
@@ -53,148 +54,120 @@ export interface ShortInfo {
     ]),
   ],
 })
-export class PropertyInfoComponent implements OnChanges, AfterContentInit {
-  @Input() propDef: ResourcePropertyDefinitionWithAllLanguages;
+export class PropertyInfoComponent implements OnInit {
+  @Input() propDef!: ResourcePropertyDefinitionWithAllLanguages;
```

Comment:
add @Input({required: true}) when a resource is a mandatory input

Reply:
Yup, agree. Will be there.